### PR TITLE
Added Brother thermal Printer

### PR DESCRIPTION
**Pull request by younessssssss**
add support for brother thermal printer:
-QR code 
-image
-barecode 
all work fine ,tested with Brother TD-4550DNWB

### CODE REVIEW

**Comment by Klemen1337** [comment]
Hi! 👋   Sorry for the wait 🙏  This is fricking awesome! But we are having code formatting issues now. I have to add some code style rules or else we are going to have a lot of merge conflicts. ☹️ 
Can you fix the merge conflicts or revert the code style changes?

**Comment by younessssssss** [comment]
hi @Klemen1337 ,i have resolved all the merge conflicts ,please tell me if there is anything to add.

**Comment by Klemen1337** [comment]
Please undo the ' to " change.

File: examples/charsets.js [hunk 1]
```diff
@@ -1,42 +1,44 @@
-const ThermalPrinter = require('../node-thermal-printer').printer;
-const Types = require('../node-thermal-printer').types;
+const ThermalPrinter = require("../node-thermal-printer").printer;
+const Types = require("../node-thermal-printer").types;
 
 async function example() {
-  const printer = new ThermalPrinter({
-    type: Types.EPSON, // 'star' or 'epson'
+  let printer = new ThermalPrinter({
```
Comment (Klemen1337) [inline examples/charsets.js:5]: Why the change to 'let' ?

**Comment by Klemen1337** [comment]
Code style change was not reverted

**Comment by geertvanleemputten** [comment]
I have a "Brother TD-4550DNWB", but somehow this is not working:
(isConnected = true and the image exists, I also get no exception while executing this code)

let printer = new ThermalPrinter({
      type: PrinterTypes.BROTHER,
      interface: "tcp://192.168.1.35:9100",
});

const isConnected = await printer.isPrinterConnected();
if (isConnected) {
      await printer.printImage("./test.png");
      printer.cut();
}
